### PR TITLE
Fix URL and default worker type in example_config

### PR DESCRIPTION
**Pull request by radzinzki**
This fixes the URL in the example_config.py

### CODE REVIEW

File: examples/example_config.py [hunk 1]
```diff
@@ -33,7 +33,8 @@
 #
 #   worker_class - The type of workers to use. The default
 #       async class should handle most 'normal' types of work
-#       loads. You'll want to read http://gunicorn/deployment.hml
+#       loads. You'll want to read
+#       http://docs.gunicorn.org/en/latest/design.html
```
Comment (berkerpeksag) [inline examples/example_config.py]: http://docs.gunicorn.org/en/latest/design.html#choosing-a-worker-type would be a better URL.

File: examples/example_config.py [hunk 2]
```diff
@@ -33,7 +33,8 @@
 #
 #   worker_class - The type of workers to use. The default
 #       async class should handle most 'normal' types of work
```
Comment (berkerpeksag) [inline examples/example_config.py]: Could also update update "async" to "sync"? Thanks!

**Comment by radzinzki** [comment]
@berkerpeksag done

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**